rust/android/Dockerfile

```diff
@@ -85,7 +85,7 @@ RUN cd /opt/openssl-3.0.1 && \
     make -j$(nproc) && make -j$(nproc) install && \
     make clean && make distclean
 
-RUN cd /opt && git clone --depth=1 --branch="dev_working" https://github.com/dannasessha/zecwallet-mobile zecwalletmobile
+RUN cd /opt && git clone --depth=1 --branch="dev" https://github.com/zingolabs/zecwallet-mobile zecwalletmobile
```

Comment:
Can we set this to `--tag="vzing-0.0.1"` I don't know if that'll work...

Reply:
I mean ...   in _place_ of `--branch="dev"`.

Reply:
But zecwallet-mobile doesn't have this tag, vzing-0.0.1 belong to zecwalletlitelib

Reply:
dev for now seems good idea to me.

Reply:
Oh shoot...   you're right...    we need to add that tag to this repo once we cut a stable release on the `main` branch of this repo, which is just a linear series of tagged commits.

I suppose it should be `vzing-0.0.1-alpha` until we get it to reliably run on both `android` and `iOS`.